src/lib.rs

```diff
@@ -653,88 +585,18 @@ pub enum LevelFilter {
     Trace,
 }
 
-// Deriving generates terrible impls of these traits
```

Comment:
This statement is not right. I created a godbolt for it. There's no real difference between derived impls and custom impls (anymore): https://rust.godbolt.org/z/7K1vjG6KE